src/github-api.js

```diff
@@ -282,10 +282,12 @@ function getGithubRepos() {
 function githubCreateRepo() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/21dcc489c3f7718bd929727b9d75c3cf1a977953/src/github-api.js#L282"></span>‘githubCreateRepo’ is defined but never used.</td>
    </tr>
  </tbody>
</table>